src/Blast.jl

```diff
@@ -7,9 +7,34 @@ using FastChebInterp
 using SpecialFunctions
 using StaticArrays
 using FFTW
+using NPZ
+using Artifacts
 
 include("projected_matter.jl")
 include("chebcoefs.jl")
 include("integrals.jl")
 
+function load_precomputed_Ts(folder; nℓ=22, nχ=96, nR=48, n_cheb=120)
```

Comment:
We know that the keyword argument you passed are necessarily `Int`. Put this information in the creation of the function, such that we can avoid problems (with people passing `Float`s).